docs/pages/access-controls/access-request-plugins/servicenow.mdx

```diff
@@ -35,6 +37,27 @@ API.
 Before making the access request ensure the user making the request has
 the `requester` role.
 
+For the plugin to know which ServiceNow rotations to check for the
+auto approval flow, the rotation you wish users to be checked against
+must be included in one of the requesting user's roles. To do so, add the `teleport.dev/schedules` annotation as shown in the following example: 
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: YOUR_ROLE_NAME_HERE
+spec:
+  allow:
+    request:
+      roles: ['editor']
+      thresholds:
+        - approve: 1
+          deny: 1
+      annotations:
+        teleport.dev/schedules:
+        - YOUR_SERVICENOW_ROTA_ID_HERE
```

Comment:
We should explain below how to determine the rotation ID.

Reply:
Added the method with the fewest steps I could find.